plugins/modules/redfish_info.py

```diff
@@ -55,6 +55,11 @@
       - Security token for authenticating to OOB controller.
     type: str
     version_added: 2.3.0
+  manager:
+    description:
+      - Name of manager on OOB controller to target.
+    type: str
+    version_added: '8.2.0'
```

Comment:
8.2.0 is already out, the next release will be 8.3.0:
```suggestion
    version_added: '8.3.0'
```